### PR TITLE
Add telega keybindings for evil

### PR DESCRIPTION
**Pull request by Vonfry**
Here is a problem about keybinds on buttons. Currently, I remove the original keybinds on these button and bind new ones which may be a reasonable solution.

### CODE REVIEW

**Comment by condy0919** [comment]
Great work

File: modes/telega/evil-collection-telega.el [hunk 1]
```diff
@@ -0,0 +1,206 @@
+;;; evil-collection-telega.el --- Evil bindings for calc -*- lexical-binding: t -*-
+
+;; Copyright (C) 2018 Pierre Neidhardt
```
Comment (condy0919) [inline modes/telega/evil-collection-telega.el:3]: Your name

File: modes/telega/evil-collection-telega.el [hunk 2]
```diff
@@ -0,0 +1,206 @@
+;;; evil-collection-telega.el --- Evil bindings for calc -*- lexical-binding: t -*-
```
Comment (condy0919) [inline modes/telega/evil-collection-telega.el:1]: calc -> telega

File: modes/telega/evil-collection-telega.el [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+;; This file is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; For a full copy of the GNU General Public License
+;; see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;; Evil bindings for calc.
```
Comment (condy0919) [inline modes/telega/evil-collection-telega.el:26]: calc -> telega

File: modes/telega/evil-collection-telega.el [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+                                        telega-chat-mode-map
+                                        telega-image-mode-map
+                                        telega-webpage-mode-map
+                                        telega-user-button-map
+                                        telega-msg-button-map
+                                        telega-chat-button-map
+                                        telega-sticker-button-map))
+
+(dolist (map evil-collection-telega-maps)
+  (eval `(defvar ,map)))
```
Comment (condy0919) [inline modes/telega/evil-collection-telega.el:42]: can it be omitted?

File: modes/telega/evil-collection-telega.el (hunk 1 above)
Comment (Vonfry) [reply modes/telega/evil-collection-telega.el]: Sorry, I forgot to modify these. Now it is done.

File: modes/telega/evil-collection-telega.el (hunk 4 above)
Comment (Vonfry) [reply modes/telega/evil-collection-telega.el:42]: Indeed, it is not necessary. However, many other modes in evil-collection have `defvar` with their maps, so I coded this. They are removed now.

**Comment by condy0919** [comment]
Thanks

**Comment by condy0919** [comment]
Thanks

**Pull request merged**